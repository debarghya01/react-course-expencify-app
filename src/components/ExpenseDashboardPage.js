import React from 'react';
import ReactDOM from 'react-dom';
import ExpensList from './ExpenseList';
import ExpenseListFilters from './ExpenseListFilters';


const ExpenseDashboardPage = () => {
    return(
        <div>
            <ExpenseListFilters />
            <ExpensList />
        </div>
    )
}

export default ExpenseDashboardPage;