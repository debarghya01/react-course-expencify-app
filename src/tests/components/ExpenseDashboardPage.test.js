import React from 'react';
import {shallow} from 'enzyme';
import ExpenseDashboardPage from '../../components/ExpenseDashboardPage';

test('testing ExpenseDashboardPage', () => {
    const wrapper = shallow(<ExpenseDashboardPage />)
    expect(wrapper).toMatchSnapshot()
})