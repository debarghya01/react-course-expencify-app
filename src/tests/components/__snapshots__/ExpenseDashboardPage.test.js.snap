// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing ExpenseDashboardPage 1`] = `
<div>
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</div>
`;
